templates/source/view.html

```diff
@@ -6,28 +6,29 @@
         <div class="jumbotron">
             <div class="row">
                 <div class="col-sm-12">
-                    <h2 class="page-title cited">
-                        <a href="{{ source.source_url }}" target="_blank">
-                            {{ source.title }}
-                        </a>
-                        {% if request.user.is_staff %}
+                    {% if request.user.is_staff %}
```

Comment:
I moved the `Edit` button to a separate `h2` element above the source title, to avoid the title and the `Edit` button running into one another. The spacing looks better for long titles now:

<img width="1214" alt="Screen Shot 2019-08-07 at 11 25 31 AM" src="https://user-images.githubusercontent.com/14170650/62722678-9bd2ce80-b9d4-11e9-9c11-7e3d61933ced.png">